src/arguments.py

```diff
@@ -27,6 +27,10 @@ def process_args(terminal_args, config_options):
         processed_args['ansible_pb'] = terminal_args.pb or config_options.get('playbooks', [])
         processed_args['ansible_inv'] = terminal_args.inventory or config_options.get('inventory', [])
         processed_args['ansible_vars'] = terminal_args.inventory or config_options.get('vars', {})
+        if terminal_args.ansible_verbosity not in [True, 1, 2, 3]:
+            raise ValueError(f"'ansible_verbosity: {terminal_args.ansible_verbosity}: must be 1, 2, or 3'")
```

Comment:
Should we account for all available verbose levels , just incase an operator wants their logs to be less verbose or go full bonkers , may be add 0,and 4 too 
Here's a breakdown of verbosity levels:
0 (default): Displays only critical information.
1 (-v): Shows basic information like task names and results.
2 (-vv): Includes more detailed output, such as variable values.
3 (-vvv): Displays additional debugging data, such as task-level operations.
4 (-vvvv): Enables connection debugging, providing a deep dive into network communication.